tools/deploy

```diff
@@ -146,6 +146,25 @@ def prepare(options: argparse.Namespace) -> None:
     clean(options)
     process(options)
 
+def log(options: argparse.Namespace) -> None:
+    check_common_options(options)
+    headers = {'key': options.key}
+    if options.lines:
+        lines = options.lines
+    else:
+        lines = None
+    payload = {'name': options.botname, 'lines': lines}
+    url = urllib.parse.urljoin(options.server, 'bots/logs/' + options.botname)
+    r = requests.get(url, json=payload, headers=headers)
+    if r.status_code == requests.codes.ok:
+        print(r.text)
+        return
+    if r.status_code == 401:
+        print('stop: Authentication error with the server. Aborting.')
```

Comment:
So far I was following the pattern of having the function name be logged with the messages in this script, so if someone simply gives us the output of this script if it fails, we could tell in which specific function it failed.

Though that'd be useful for chained commands like `prepare`, but not much relevant here except for consistency. Thoughts?

Reply:
My bad, I forgot to change it. Mentioning function name is good for debugging purpose for users.Since we are not going to put logs in prepare command, we can get away with the function name and simply show the error but it will be helpful if we mention the function name.
I'll change stop to logs.